Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -248,20 +261,22 @@ public void centerConeStack(RobotCameraPipeline pipeline) {
         double xOffset, width;
 
         do {
-            xOffset = pipeline.xPosition - Constants.CAMERA_CENTER_X;
+            xOffset = 400-pipeline.xPosition;
             width = pipeline.width;
 
             // drive forward while centering on the cone stack if contour exists
             if (width == 0) {
                 break;
             } else {
-                driveWithIMU(coneStackPixelsMotorPower(xOffset), coneStackWidthMotorPower(width), 0.0);
-                telemetry.addData("xMotorPower", coneStackPixelsMotorPower(xOffset));
+                //driveWithIMU(coneStackPixelsAndWidthToStrafingPower(xOffset, width), coneStackWidthMotorPower(width), 1.09*coneStackPixelsAndWidthToTurningPower(xOffset, width));
+                telemetry.addData("xStrafingPower", coneStackPixelsAndWidthToStrafingPower(xOffset, width));
+                telemetry.addData("xTurningPower", coneStackPixelsAndWidthToTurningPower(xOffset, width));
                 telemetry.addData("yMotorPower", coneStackWidthMotorPower(width));
```

Comment:
If you're going to call these methods in your code and in your telemetry, it's better to use a variable for speed if the method is slow and to ensure that you are actually printing out the same value that you are using in your code.

Reply:
Resolved, made them use the same variable